ci-operator/config/openshift/console/openshift-console-release-4.7.yaml

```diff
@@ -7,7 +7,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: tectonic-console-builder-v21
+    tag: tectonic-console-builder-v22
```

Comment:
Oops, we shouldn't have updated 4.7 since it doesn't have the webpack-cli change. I'll open a follow on PR to change this.